### PR TITLE
fix(drawer): animation

### PR DESCRIPTION
**Pull request by acateland**
**Please check if the PR fulfills these requirements**

- [X] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?**

- [X] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)
Drawer started to appear on right at 40% of the animation, and with like 10% of the drawer visible, same thing when closing, making the animation feel choppy.
the transition triggered repaint instead of using gpu layer rendering

**What is the new behavior?**
right position animation animation is now replaced by transform animation
use chrome console rendering and animation tool to check the difference.

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: src/Layout/Drawers/Drawers.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // Drawers styles
 
-$tc-drawer-initial-offset: 1000px !default;
+$tc-drawer-initial-offset: 100% !default;
```
Comment (jsomsanith) [inline src/Layout/Drawers/Drawers.scss]: Does this var really needed ? Can we imagine an animation only from N% to 0 ?

File: src/Layout/Drawers/Drawers.scss [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 $tc-drawer-bgcolor: #FFF !default;
 $tc-drawer-transition-duration: 1s !default;
 $tc-drawer-transition-easing: ease-in !default;
```
Comment (jsomsanith) [inline src/Layout/Drawers/Drawers.scss]: 1s ease-in feels veeeery slooow :)
I would recommand 0.75s ease-out. A fast beginning and a smooth landing feels better IMHO.

Comment (acateland) [inline src/Layout/Drawers/Drawers.scss]: i tested some animation speed
and 230ms with custom cubic bezier look quite nice

File: src/Layout/Drawers/Drawers.scss (hunk 1 above)
Comment (acateland) [inline src/Layout/Drawers/Drawers.scss]: you are right

**Pull request merged**